modules/ROOT/pages/fd-project-history.adoc

```diff
@@ -0,0 +1,27 @@
+= Discover What’s Changed in a Project
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords:
+
+:imagesdir: ../assets/images
+
+You can see a history of changes you make to a project in Flow Designer.
+The saved states are listed by time/date and activity, and their name depends on the activity you were doing at the time the save occurred.
```

Comment:
It also includes the person that made the change.

Reply:
Fixed.